executors/http/executor.go

```diff
@@ -54,7 +54,7 @@ func (Executor) GetDefaultAssertions() venom.StepAssertions {
 }
 
 // Run execute TestStep
-func (Executor) Run(l *log.Entry, aliases venom.Aliases, step venom.TestStep) (venom.ExecutorResult, error) {
+func (Executor) Run(l *log.Entry, aliases venom.Aliases, step venom.TestStep, templater *venom.Templater) (venom.ExecutorResult, error) {
```

Comment:
maybe we should apply templater on all step params ?
it's a bit weird to have to handle this in all executor.